releases/reference-assemblies/README.md

```diff
@@ -69,7 +46,7 @@ Add the `TargetFramework` property and a `PackageReference` like in the followin
     <TargetFramework>net472</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-alpha-5" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" PrivateAssets="All" />
```

Comment:
I'm not sure if the advise here is to add a package reference to this, I believe this should be automatically added by the SDK, so I'm not sure why you would want to manually add the reference yourself.

Reply:
That was talked about, but as I understand it needs to be added manually. Perhaps the SDK delivered version will land in .NET 5?

Reply:
Yes, you still need to manually add the reference.  Hopefully in the .NET 5 SDK we will add the feature to reference these automatically if necessary.